packages/player/src/components/shortcut.tsx

```diff
@@ -74,6 +74,15 @@ export const Shortcut: React.FC<ShortcutProps> = ({
 
   const shortCutsRef = React.useRef<any[]>([])
 
+  React.useEffect(() => {
+    document.addEventListener('fullscreenchange', (e) => {
+      const isFullscreenOff = false === !!document.fullscreenElement
+      if (isFullscreenOff) {
+        videoService.send({type: 'EXIT_FULLSCREEN', element: rootElem})
+      }
+    })
+  }, [])
+
```

Comment:
the code above is needed to listen the exit from fullscreen by clicking on escape button and fire related handler. Without that the state didn't get toggled.